File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self,arr: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        n=len(arr)
        for i in range(len(arr)):
            for j in range(0,len(arr)-1):
                if(arr[j]>arr[j+1]):
                    arr[j],arr[j+1]=arr[j+1],arr[j]
        return arr      

File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int temp;
        int a=nums.size();
        for(int i=0;i<a;i++)
        {
          for(int j=0;j<a-1;j++)
          {
              if(nums[j]>nums[j+1])
              {
                temp=nums[j];
                nums[j]=nums[j+1];
                nums[j+1]=temp;
              }
              
          }
        }

    }
};